scripts/zones/Inner_Horutoto_Ruins/mobs/Nocuous_Weapon.lua

```diff
@@ -0,0 +1,42 @@
+-----------------------------------
+-- Area: Inner Horutoto Ruins
+--  NM: Nocuous Weapon
+-----------------------------------
+require("scripts/globals/status");
+require("scripts/globals/magic");
+require("scripts/globals/msg");
+
+function onMobInitialize(mob)
+    mob:setMobMod(MOBMOD_ADD_EFFECT,mob:getShortID());
+end;
+
+function onAdditionalEffect(mob, player)
+    local chance = 25;
+    local resist = applyResistanceAddEffect(mob,player,ELE_WATER,EFFECT_POISON);
+    if (math.random(0,99) >= chance or resist <= 0.5) then
+        return 0,0,0;
+    else
+        local duration = 30;
+        if (mob:getMainLvl() > player:getMainLvl()) then
+            duration = duration + (mob:getMainLvl() - player:getMainLvl())
+        end
+        duration = utils.clamp(duration,1,45);
+        duration = duration * resist;
+        if (not player:hasStatusEffect(EFFECT_POISON)) then
+            player:addStatusEffect(EFFECT_POISON, 1, 3, duration); -- Don't know potency on the poison.
+        end
+        return SUBEFFECT_POISON, msgBasic.ADD_EFFECT_STATUS, EFFECT_POISON;
+    end
+end;
+
+function onMobDeath(mob, player, isKiller)
+end;
+
+function onMobDespawn(mob)
+    SetServerVariable("[POP]Nocuous_Weapon", os.time() + 3600); -- 1 hour
+    DisallowRespawn(mob:getID(), true);
```

Comment:
Sorry we didn't send you the memo: we switched to using a listener in the placeholders now, but haven't updated all the old scripts yet https://github.com/DarkstarProject/darkstar/commit/5c06b263b421310989184d497f8773a75c6282f8#diff-e1863da932c3fbbf882f221b398ae513R42

Reply:
Understood.  Hold this PR open and I'll make the switch tomorrow.